Library/Homebrew/cmd/options.rb

```diff
@@ -40,15 +38,7 @@ def options
     elsif args.installed?
       puts_options Formula.installed.sort, args: args
     elsif !args.command.nil?
-      path = Commands.path(args.command)
-      odie "Unknown command: #{args.command}" unless path
```

Comment:
Is `odie` still needed?

```rb
odie "Unknown command: #{args.command}" if cmd_options.nil?
```

Note: `[].nil?` returns `false`, but `[].blank?` returns `true`

Reply:
You're right that the `odie` should still be here. To clarify, you're suggesting this, right?

```ruby
    elsif args.command.present?
      cmd_options = Commands.command_options(args.command)
      odie "Unknown command: #{args.command}" if cmd_options.nil?
      ...
```

Reply:
Yep! 👍